spec/amber/server/env_spec.cr

```diff
@@ -4,40 +4,85 @@ describe Amber do
   {% for env in %w(development staging test sandbox production) %}
     describe ".{{env.id}}?" do
       it "returns true when the environment is {{env.id}}" do
-        Amber::Settings.env = {{env}}
-        Amber.env.{{env.id}}?.should be_truthy
+        amber_env = Amber::Env.new {{env}}
+        amber_env.{{env.id}}?.should be_truthy
       end
 
       it "returns false when the environment does not match" do
-        Amber::Settings.env = "invalid environment"
-        Amber.env.{{env.id}}?.should be_falsey
+        amber_env = Amber::Env.new "invalid environment"
+        amber_env.{{env.id}}?.should be_falsey
       end
     end
   {% end %}
 
+  describe "Amber.env" do
```

Comment:
Can you move this out into its own test. This can be move to the amber_spec file. 

Describe Amber do
   Describe “.env” do 
      ....
   End 
End

Reply:
Yeah.